src/MCPClient/lib/clientScripts/normalizeReport.py

```diff
@@ -119,19 +120,24 @@ def report(uuid):
         except IndexError:
             logger.info('No normalization failures have been detected in type "%s"', jobtype)
             continue
-        tasks = Task.objects.filter(Q(job=job) & ~Q(exitcode=0))
-        if not tasks.count():
+        tasks = Task.objects.filter(job=job).exclude(exitcode=0)
+        if not tasks.exists():
             logger.info('No normalization failures have been detected in type "%s"', jobtype)
             continue
-        failed_tasks[jobtype] = tasks
+        failed_tasks[jobtype] = tasks.values('filename', 'fileuuid', 'exitcode')
+        for item in failed_tasks[jobtype]:
+            try:
+                item['location'] = File.objects.get(uuid=item['fileuuid']).currentlocation.replace('%SIPDirectory%', '')
+            except File.DoesNotExist:
+                pass
 
     if not len(failed_tasks):
         logger.info('Normalization report is not being sent because no failures have been detected.')
-        return 0
```

Comment:
Did you mean to delete this? If there's no failed tasks we probably want an early abort.